App.js

```diff
@@ -1,10 +1,18 @@
+/**
+ * Create App Page
+ * by id.ly Team
+ */
+
+//Import Libraries
 import React, { Component } from 'react';
-import { Platform } from 'react-native';
+import { Platform, YellowBox } from 'react-native';
 import { Provider } from 'react-redux';
 import { Font } from 'expo';
 import store from './app/store';
 import Main from './app/index';
 
+console.ignoredYellowBox = ['Warning: componentWill'];
```

Comment:
@joewetton Only "Invalid Child" and "Key Prop" were found after I ignored "Waring: componentWill" and merged our latest master. By the way, we are able to see any warning so far, expect a warning with "Waring: componentWill", so just make sure that we don't use "componentWill*" at future develpment, it will be fine.